src/Makefile

```diff
@@ -736,7 +736,7 @@ clean: objclean profileclean
 	@rm -f .depend *~ core
 
 net:
-	$(eval nnuenet := $(shell grep EvalFile ucioption.cpp | grep Option | sed 's/.*\(nn-[a-z0-9]\{12\}.nnue\).*/\1/'))
+	$(eval nnuenet := $(shell grep EvalFileDefaultName evaluate.h | sed 's/.*\(nn-[a-z0-9]\{12\}.nnue\).*/\1/'))
```

Comment:
probably good to make it `$(shell grep EvalFileDefaultName evaluate.h |  grep define | sed 's/.*\(nn-[a-z0-9]\{12\}.nnue\).*/\1/')) ` so it only matches one line, not every line with the variable name.